app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -23,7 +31,52 @@ class FragmentMoviesDetails : Fragment() {
                 }
             }
 
+        view.findViewById<TextView>(R.id.age_limit)
+                .text = context?.getString(R.string.age_limit, 13.toString())
+
+        view.findViewById<TextView>(R.id.reviews_count)
+                .text = context?.getString(R.string.reviews, 125.toString())
+
         return view
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        recycler = view.findViewById(R.id.actor_list)
+        recycler?.apply {
+            layoutManager = LinearLayoutManager(context, RecyclerView.HORIZONTAL, false)
+
+            adapter = ActorsAdapter()
+
+            // add ItemDecoration (spaces between items)
+            val itemDecorator = DividerItemDecoration(context, DividerItemDecoration.HORIZONTAL)
```

Comment:
I'm not sure if I would go with item decorator or just view holder margin

Reply:
Minor 👆

Reply:
I needed separators between the actors in the list, so I used the ready-made DividerItemDecoration

Reply:
It was possible to use margins, for example on the right, but decided to try a decorator.